src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.Locale;
+import java.util.NoSuchElementException;
```

Comment:
The implementations section only shows 2 out of the 5 features stated in your DG.
Should you include the implementations of all your features?
![image](https://user-images.githubusercontent.com/60385689/97514433-a0184c80-19c9-11eb-8ee8-a6fc42bd5dd4.png)